bundles/pixi.js-node/src/index.ts

```diff
@@ -0,0 +1,126 @@
+import './adapter';
+
+import { Application } from '@pixi/app';
+import { BatchRenderer, extensions, INSTALLED } from '@pixi/core';
+import { Extract } from '@pixi/extract';
+import { AlphaFilter } from '@pixi/filter-alpha';
+import { BlurFilter, BlurFilterPass } from '@pixi/filter-blur';
+import { ColorMatrixFilter } from '@pixi/filter-color-matrix';
+import { DisplacementFilter } from '@pixi/filter-displacement';
+import { FXAAFilter } from '@pixi/filter-fxaa';
+import { NoiseFilter } from '@pixi/filter-noise';
+import '@pixi/mixin-cache-as-bitmap';
+import '@pixi/mixin-get-child-by-name';
+import '@pixi/mixin-get-global-position';
+import { ParticleRenderer } from '@pixi/particle-container';
+import { Prepare } from '@pixi/prepare';
+import { TilingSpriteRenderer } from '@pixi/sprite-tiling';
+import { TickerPlugin } from '@pixi/ticker';
+import * as utils from '@pixi/utils';
+// eslint-disable-next-line @typescript-eslint/no-duplicate-imports
+import { loadNodeBitmapFont, loadNodeFont, loadNodeTexture, NodeCanvasResource } from './adapter';
+import { loadBitmapFont, loadTextures, loadWebFont } from '@pixi/assets';
+
+// Remove the default loader plugins
+extensions.remove(
+    loadTextures,
+    loadWebFont,
+    loadBitmapFont,
+);
+extensions.add(
+    // Install renderer plugins
+    Extract,
+    ParticleRenderer,
+    Prepare,
+    BatchRenderer,
+    TilingSpriteRenderer,
+
+    // Install application plugins
+    TickerPlugin,
+
+    // Install loader plugins
+    loadNodeTexture,
+    loadNodeFont,
+    loadNodeBitmapFont
+);
+
+// reset installed resources and remove resize plugin from Application
+INSTALLED.length = 0;
+INSTALLED.push(NodeCanvasResource);
+Application['_plugins'].shift(); // eslint-disable-line dot-notation
```

Comment:
A better way to remove the ResizePlugin is to export it from `@pixi/app` and use the extensions.remove. Much better and more explicit than shifting.